src/main/java/pl/kzcwat/localincidentserver/region/request/RegionRequest.java

```diff
@@ -0,0 +1,16 @@
+package pl.kzcwat.localincidentserver.region.request;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Data
+@Builder
+public class RegionRequest {
+    @NotNull
+    @Size(min = 3)
+    private String regionName;
```

Comment:
Tu znów się naciąłem jak testowałem teraz w Postmanie😉 wystarczy name, `region` wynika z kontekstu

```suggestion
    private String name;
```

Reply:
Rozwiązano w 55cfd9c29baf1facb022469b315842b71d6b0af3